src/main/resources/application.properties

```diff
@@ -4,7 +4,7 @@ spring.datasource.username=nika.avalishvili
 spring.datasource.password=#REF!123
 #spring.datasource.driverClassName=org.postgresql.Driver
 spring.jpa.show-sql=true
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
I don't think we need `update` here
It should `none` since we manage db through liquibase scripts

Reply:
I broke something and it was unsuccessful solution. I reverted none back. Still trying to find solution and will upload updated code very soon, hope.